lib/ui/discover/discover_page.dart

```diff
@@ -16,33 +94,103 @@ class HomeDiscoverPage extends StatefulWidget {
+        return _buildDiscoverPage(context, viewModel.articleDirectory.articles,
+            viewModel.getRelatedArticles, discoverStyle);
       case LoadingStatus.ERROR:
         return Text(Strings.errorString);
     }
   }
 }
 
-Widget _buildList(BuildContext context, List<ArticleEntity> articlesList, Function getRelatedArticles) {
-  return ListView(
-      padding: const EdgeInsets.only(top: 20.0),
-      children: (articlesList != null) ? (articlesList.map((article) =>
-          DiscoveryListItem().buildListItem(article, getRelatedArticles))).toList() : [Container()]);
-}
+  Widget _buildDiscoverPage(BuildContext context, List<ArticleEntity> articlesList, Function getRelatedArticles, HomeDiscoverPageStyle discoverStyle) {
+    return ListView(
+      children: <Widget>[
+        _buildScreenTitle(discoverStyle),
+        _buildHeaderArticle(articlesList.first, discoverStyle),
+        _buildArticlesList(articlesList, getRelatedArticles, discoverStyle)
+      ],
+    );
+  }
+
+  Widget _buildScreenTitle(HomeDiscoverPageStyle discoverStyle) {
+    return Container(
+      padding: discoverStyle.titlePadding,
+      child: Row(
+        crossAxisAlignment: discoverStyle.leftAlignmentHorizontal,
+        children: <Widget>[
+          Text(Strings.discoverTab, style: discoverStyle.titlePageStyle)
+        ],
+      ),
+    );
+  }
+
+  Widget _buildHeaderArticle(ArticleEntity firstArticle, HomeDiscoverPageStyle discoverStyle) {
+    return Column(
+          children: <Widget>[
+            Container(
+              padding: discoverStyle.headerArticlePadding,
+              child: Column(
+                crossAxisAlignment: discoverStyle.leftAlignmentHorizontal,
+                children: <Widget>[
+                  _buildHeaderArticleImage(firstArticle, discoverStyle),
+                  SizedBox(height: discoverStyle.spaceBetweenHeaderImageAndText),
+                  Text(
+                      firstArticle.title,
+                      maxLines: discoverStyle.headerTitleMaxLines,
+                      style: discoverStyle.titleArticleStyle
+                  ),
+                  SizedBox(height: discoverStyle.spaceBetweenTitleAndSummary),
+                  Text(
+                      firstArticle.summary,
+                      maxLines: discoverStyle.headerSummaryMaxLines,
+                      style: discoverStyle.summaryArticleStyle
+                  )
+                ],
+              ),
+            ),
+            buildListSeparator(discoverStyle)
+          ],
+        );
+  }
+
+  Widget buildListSeparator(HomeDiscoverPageStyle discoverStyle) {
+    return Container(
+        height: discoverStyle.separatorHeight,
+        color: discoverStyle.separatorColor,
+        margin: discoverStyle.separatorIndentation
+    );
+  }
 
+  Widget _buildArticlesList(List<ArticleEntity> articlesList, Function getRelatedArticles, HomeDiscoverPageStyle discoverStyle) {
```

Comment:
let push the responsibility of repeated article fetch to the article detail widget not this list widget.